components/ws-proxy/pkg/proxy/workspacerouter_test.go

```diff
@@ -204,34 +193,37 @@ func TestMatchWorkspaceHostHeader(t *testing.T) {
 			},
 		},
 		{
-			Name:       "mini browser workspace match",
-			HostHeader: "browser-amaranth-smelt-9ba20cc1" + wsHostSuffix,
+			Name:       "unique webview workspace match 2",
+			HostHeader: "0d9rkrj560blqb5s07q431ru9mhg19k1k4bqgd1dbprtgmt7vuhk" + wsHostSuffix,
+			Path:       "/amaranth-smelt-9ba20cc1/index.html",
 			Expected: matchResult{
 				MatchesWorkspace: true,
 				WorkspaceVars: map[string]string{
-					foreignOriginIdentifier: "browser-",
 					workspaceIDIdentifier:   "amaranth-smelt-9ba20cc1",
+					foreignOriginIdentifier: "0d9rkrj560blqb5s07q431ru9mhg19k1k4bqgd1dbprtgmt7vuhk",
+					foreignPathIdentifier:   "/index.html",
 				},
 			},
 		},
 		{
-			Name:       "port match",
-			HostHeader: "8080-amaranth-smelt-9ba20cc1" + wsHostSuffix,
+			Name:       "unique webworker workspace match",
+			HostHeader: "v--0d9rkrj560blqb5s07q431ru9mhg19k1k4bqgd1dbprtgmt7vuhk" + wsHostSuffix,
+			Path:       "/amaranth-smelt-9ba20cc1/index.html",
 			Expected: matchResult{
-				MatchesPort: true,
-				PortVars: map[string]string{
+				MatchesWorkspace: true,
+				WorkspaceVars: map[string]string{
 					workspaceIDIdentifier:   "amaranth-smelt-9ba20cc1",
-					workspacePortIdentifier: "8080",
+					foreignOriginIdentifier: "v--0d9rkrj560blqb5s07q431ru9mhg19k1k4bqgd1dbprtgmt7vuhk",
+					foreignPathIdentifier:   "/index.html",
 				},
 			},
 		},
 		{
-			Name:       "webview port match",
-			HostHeader: "webview-8080-amaranth-smelt-9ba20cc1" + wsHostSuffix,
+			Name:       "port match",
```

Comment:
i think this one should just go, port location should be already covered by other tests

Reply:
ah, actually, it is test for webviews on port location for dev setup, how does it work now?

Reply:
Added test for webviews on port location just in case it really shouldn't matter webworker runs on same origin in dev and webviews we could say it's useful but we can just force it to run from same origin too (we are changing webview code either way)